sgt-back-end/queries.js

```diff
@@ -0,0 +1,17 @@
+const doesGradeIdExist = 'SELECT * FROM "grades" WHERE "gradeId" = $1';
+const UPDATE = 'UPDATE "grades" SET "name" = $2,"course" = $3,"score" = $4 WHERE "gradeId" = $1 RETURNING *';
+const INSERT = `
+  INSERT INTO GRADES (name,course,score)
+  VALUES($1,$2,$3)
+  `;
+const GET = 'SELECT * FROM GRADES';
+const DELETE = 'DELETE FROM "grades" WHERE "gradeId" = $1';
+
```

Comment:
As I said above, I like having these all in one file. However, I would name the variables according to what they accomplish instead of according to what SQL command they issue. Something like: `GET_GRADE`, `GET_ALL_GRADES`, `INSERT_GRADE` and so on.